pkg/security/activitydump/activity_dump.go

```diff
@@ -710,6 +710,25 @@ func (ad *ActivityDump) FindMatchingNodes(comm string) []*ProcessActivityNode {
 	return res
 }
 
+// GetImageNameTag returns the image name and tag for the profiled container
+func (ad *ActivityDump) GetImageNameTag() (string, string) {
```

Comment:
I think we can make use of this function here: https://github.com/DataDog/datadog-agent/blob/paulcacheux/secprofiles-diff/pkg/security/activitydump/manager.go#L187

Reply:
There is a difference in the locking behavior, postponed for now